sass/components/_ciq-chart-table.scss

```diff
@@ -16,147 +12,201 @@
+                color: themed('ChartTableBodyHeadText');
+            }
 
-            &:hover {
-                @include themify($themes) {
-                    background: themed('ChartTableHeadHover');
-                }
+            &.sticky-top,
+            &.sticky-bottom {
+                z-index: 8;
+            }
+            &.sticky-bottom {
+                position: absolute;
+                top: auto !important;
+                bottom: 0px !important;
             }
-            svg {
-                width: 16px;
-                height: 16px;
-                stroke: none;
+            &.sticky-top {
+                position: fixed;
+                top: 0px;
             }
         }
     }
-    tbody {
-        display: block;
-    }
-    .cq-change-cell {
-        display: inline-flex;
-    }
-}
-
-.ciq-table-head {
-    width: var(--table-width);
-    display: block;
-    @include themify($themes) {
-        background: themed('ChartTableHeadBg');
-        color: themed('ChartTableHeadText');
-        border-bottom: 1px solid themed('ChartTableHeadBorder');
-    }
-
-    .ciq-table-cell {
-        font-size: 14px;
-        font-weight: bold;
-        font-style: normal;
-        line-height: 1.43;
-        letter-spacing: normal;
-        padding: 16px 5px;
-    }
 }
 
-.ciq-table-row {
+.ciq-chart-table {
+    width: 100%;
+    border-collapse: collapse;
+    table-layout: fixed;
     @include themify($themes) {
-        border-bottom: 1px solid themed('ChartTableBodyBorder');
+        background: themed('ChartTableBg');
     }
 
-    .ciq-table-cell {
-        padding: 16px 5px;
-        font-size: 14px;
-        font-weight: normal;
-        font-style: normal;
-        line-height: 1.43;
-        letter-spacing: normal;
+    tr {
         @include themify($themes) {
-            color: themed('ChartTableBodyText');
+            border-bottom: 1px solid themed('ChartTableBodyBorder');
         }
 
-        .cq-change:after {
-            margin: 3px 0 0 5px;
+        &:last-child {
+            border-bottom: none;
         }
-        .up {
+        th,td {
+            text-align: right;
+            box-sizing: border-box;
+            padding: 16px 5px;
+            font-size: 14px;
+            font-weight: normal;
+            font-style: normal;
+            line-height: 1.43;
+            letter-spacing: normal;
             @include themify($themes) {
-                color: $COLOR_GREEN_1;
+                color: themed('ChartTableBodyText');
+            }
+
+            .cq-change:after {
+                margin: 3px 0 0 5px;
+            }
+            .up {
+                @include themify($themes) {
+                    color: $COLOR_GREEN_1;
+                }
+            }
+            .down {
+                @include themify($themes) {
+                    color: $COLOR_RED;
+                }
+            }
+            &.before-last-child {

```

Comment:
you could use :nth-last-child(2) to get the one before last child if you need it, but here and the following nth-childs, why not using classes like this to prevent indexing? the table is not dynamic, although this is a small component and doesn't have a huge impact so it's your call